payments/payment-methods/adyen/src/main/java/com/appcoins/payment_method/adyen/presentation/AdyenActionResolveContract.kt

```diff
@@ -0,0 +1,44 @@
+package com.appcoins.payment_method.adyen.presentation
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Parcelable
+import androidx.activity.result.contract.ActivityResultContract
+import com.adyen.checkout.components.ActionComponentData
+import com.appcoins.payment_method.adyen.presentation.ActionResolution.Success
+
+class AdyenActionResolveContract<T : Parcelable>(private val resolver: Class<*>) :
```

Comment:
Shouldn't the Class type be `out Activity`

Reply:
fixed